weblate/templates/widgets.html

```diff
@@ -22,110 +22,76 @@ <h2>{% translate "Promoting translation projects" %}</h2>
-                <textarea class="code-example form-control">
-.. image:: {{ color.url }}
-    :alt: {% translate "Translation status" %}
-    :target: {{ engage_url }}</textarea>
-              {% endfor %}
-            </div>
-
-            <div class="tab-pane" id="{{ widget.name }}-textile-code">
-              {% for color in widget.colors %}
-                <textarea class="code-example form-control">
-!{{ color.url }}!:{{ engage_url }}</textarea>
-              {% endfor %}
-            </div>
-
-            <div class="tab-pane" id="{{ widget.name }}-url-code">
-              {% for color in widget.colors %}
-                <textarea class="code-example form-control">
-{{ color.url }}</textarea>
-              {% endfor %}
-            </div>
-
-          </div>
-
+        </select>
+        <label for="translation_language" class="form-label mt-2">{% translate "Language" %}</label>
+        <select id="translation_language" class="select-tab form-control">
+          {% for language in form.lang.field.choices %}
+            <option value="{{ language.0 }}">{% translate language.1 %}</option>
+          {% endfor %}
+        </select>
+        <label for="component" class="form-label mt-2">{% translate "Component" %}</label>
+        <select id="component" class="select-tab form-control">
+          {% for component in form.component.field.choices %}
+            <option value="{{ component.0 }}">{% translate component.1 %}</option>
+          {% endfor %}
+        </select>
+        <label for="code_language" class="form-label mt-2">{% translate "Code" %}</label>
+        <select id="code_language" class="select-tab form-control">
+          <option value="html">{% translate "HTML" %}</option>
+          <option value="bb-code">{% translate "BBCode" %}</option>
+          <option value="mdk">{% translate "Markdown" %}</option>
+          <option value="rst">{% translate "reStructuredText" %}</option>
+          <option value="textile-code">{% translate "Textile code" %}</option>
+          <option value="url">{% translate "Image URL" %}</option>
+        </select>
+        <label for="color_select" class="form-label mt-2">{% translate "Color" %}</label>
+        <select id="color_select" class="form-control">
+          <!-- Options will be populated dynamically -->
+        </select>
+        <div id="extra-parameters"></div>
+      </div>
+    </div>
+
+    <div class="col-md-8">
+      <div class="widget-card p-3">
+        <h4>{% translate "Live Preview" %}</h4>
+        <div class="preview-box" id="preview">
+          <img id="widget-image"
+               src="data:,"
+               alt="{% translate "Translation status" %}"
+               style="max-width: 500px" />
         </div>
+      </div>
 
+      <div class="widget-card p-3">
+        <h4 id="embedCodeLabel">{% translate "Embed Code" %}</h4>
+        <textarea id="embedCode" aria-labelledby="embedCodeLabel" class="code-example form-control" rows="3" readonly>
```

Comment:
This could also be a div with a copy button, something similar to what we do with SSH keys:

https://github.com/WeblateOrg/weblate/blob/19908677f12e01fd216855968886616f9ef196b0/weblate/templates/snippets/ssh-key.html#L12-L22

Reply:
Added.